webplugin/js/app/zendesk-chat-service.js

```diff
@@ -85,4 +103,33 @@ function ZendeskChatService() {
             },
         });
     };
+
+    _this.handleZendeskAgentFileSendEvent= function (event) {
+        window.console.log("handleZendeskAgentFileSendEvent ",event);
+
+        var messagePxy = {
+            fileAttachment: event.attachment,
+            fromUserName: event.nick.split(":")[1],
```

Comment:
what is 'nick'?

Reply:
When a Zendesk agent sends a message, we get this 'nick' property in the event object. This 'nick' property contains their zendesk-ID, that we use in server side to fetch/store their details from the DB.
<img width="336" alt="image" src="https://user-images.githubusercontent.com/36797892/147722805-15d3fd4f-87c2-4af4-ba3f-d3393c4e05d6.png">